Foodiea/Models/Manager/APIManager.m

```diff
@@ -56,4 +73,30 @@ - (PFFileObject *)getPFFileFromImage: (UIImage * _Nullable)image {
     return [PFFileObject fileObjectWithName:@"image.png" data:imageData];
 }
 
+#pragma mark - keys
+
+- (NSString *)getGoogleKey{
+    NSString *path = [[NSBundle mainBundle] pathForResource: @"Keys" ofType: @"plist"];
+    NSDictionary *dict = [NSDictionary dictionaryWithContentsOfFile: path];
+    NSString *key = [dict objectForKey: @"GOOGLE_API_KEY"];
+    
+    return key;
+}
```

Comment:
Ideally you would either:
- Put this google function into it own Google config/manger file
- OR put all of these key's and app id functions into a config class